packages/contracts-bedrock/src/safe/DeputyPauseModule.sol

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Safe
+import { GnosisSafe as Safe } from "safe-contracts/GnosisSafe.sol";
+import { Enum } from "safe-contracts/common/Enum.sol";
+
+// Contracts
+import { EIP712 } from "@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol";
```

Comment:
Any idea why the file is called draft?

Reply:
Switching to 0.8.25 which doesn't have this as draft